core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1404,4 +1404,13 @@ private SqlLibraryOperators() {
           writer.endList(parenthesisFrame);
         }
       };
+
+  @LibraryOperator(libraries = {TERADATA})
+  public static final SqlFunction REGEXP_SIMILAR =
```

Comment:
As mentioned ih the MR ,please donot add teradata specific function here

Reply:
Done. Instead of REGEXP_SIMILAR fun added REGEXP_LIKE() function ( which is present in MySQL,ORACLE & SNOWFLAKE  dialects)